src/GraphQLBackend.ts

```diff
@@ -85,27 +86,39 @@ export class GraphQLBackendCreator {
    * Create backend with all related resources
    */
   public async createBackend(): Promise<IGraphQLBackend> {
-    const backend: IGraphQLBackend = {};
-
+    let backend: IGraphQLBackend = {};
+    
     try {
       await this.schemaParser.build(this.config);
       const context = this.schemaParser.getContext()
 
       if (this.resolverManager) {
-        backend.resolvers = await this.resolverManager.build(this.dbContextProvider, context.types, this.resolverTypes);
+        this.resolvers = await this.resolverManager.build(this.dbContextProvider, context.types, this.resolverTypes);
       } else {
         logger.info("Resolver generation skipped.")
       }
 
       if (this.config.generateGraphQLSchema) {
         logger.info("Generating schema")
         const generator = new GraphQLSchemaGenerator();
-        const outputSchema = generator.generateNewSchema(context, backend.resolvers, this.config);
+        const outputSchema = generator.generateNewSchema(context, this.resolvers, this.config);
         backend.schema = outputSchema;
       } else {
         logger.info("Schema generation skipped.")
       }
 
+      backend.resolvers = {}
+      backend.resolvers["Query"] = {}
+      backend.resolvers["Mutation"] = {}
+      for (const value of this.resolvers) {
```

Comment:
Clean Code. Responsibility. This should be in resolvers module. Really we should avoid format.of data being wrong in the first place